contributions/demo/yuxin-urama/Readme.md

```diff
@@ -0,0 +1,14 @@
+# Video Demo: Github integration with Jenkins
+
+## Members
+Ifeoma Urama (Urama@kth.se)
+
+Yuxin Meng (yuxinm@kth.se)
+
+## Outline
+1. Introduction of Jenkins as DevOps tool.
+2. Why use Jenkins with Github.
+3. Demo on how to integrate git with Jenkins
```

Comment:
Make sure to explain the details and show an actual Github-Jenkins setup up and running.